### PR TITLE
Update placeholders without 'translate'

### PR DESCRIPTION
**Pull request by luixxiul**
This PR intends to update placeholders without `translate` to make them translatable on Transifex. Please note that it is required to pull the changes to Transifex manually.

If these strings have been intentionally made untranslatable due to any reasons, please let me know, so that I would close the PR.

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

### CODE REVIEW

File: Duplicati/Server/webroot/ngax/templates/commandline.html [hunk 1]
```diff
@@ -14,22 +14,22 @@
 
             <div class="input textarea linklabel">
                 <label for="target"><a href class="target" ng-click="EditUriState = true">Target URL &gt;</a></label>
-                <textarea name="target" id="target" ng-model="TargetURL" placeholder="Enter a url, or click the 'Target URL &gt;' link"></textarea>
+                <textarea name="target" id="target" ng-model="TargetURL" placeholder="{{'Enter a url, or click the \'Target URL &gt;\' link | translate}}"></textarea>
```
Comment (luixxiul) [inline Duplicati/Server/webroot/ngax/templates/commandline.html:17]: I am not perfectly sure if escaping would work this way... Fixing it if needed is really appreciated.

Comment (kenkendk) [reply Duplicati/Server/webroot/ngax/templates/commandline.html:17]: Looks like this is what it looks like the other areas have done, so I think it is fine.

**Comment by kenkendk** [comment]
Merged and pushed to Transifex

**Comment by luixxiul** [comment]
Thank you!

**Pull request merged**